eloquent/ci-nightly-performance-prerequisites.yaml

```diff
@@ -0,0 +1,105 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  CMAKE_PREFIX_PATH: '/opt/ros/melodic:$CMAKE_PREFIX_PATH'
+  LD_LIBRARY_PATH: '/opt/ros/melodic/lib:$LD_LIBRARY_PATH'
+  PYTHONPATH: '/opt/ros/melodic/lib/python2.7/dist-packages:$PYTHONPATH'
+  ROS_PACKAGE_PATH: '/opt/ros/melodic/share'
+  ROS_PYTHON_VERSION: '3'
+build_tool: colcon
+build_tool_args: '--cmake-args -DCMAKE_BUILD_TYPE=Release --no-warn-unused-cli'
+install_packages:
+- default-jdk # for CycloneDDS
+- libasio-dev # for FastRTPS
+- libtinyxml2-dev # for FastRTPS
+- maven # for CycloneDDS
+- ros-melodic-common-msgs
+- ros-melodic-rosbash
+- ros-melodic-roscpp
+- ros-melodic-roscpp-tutorials
+- ros-melodic-roslaunch
+- ros-melodic-rosmsg
+- ros-melodic-rospy-tutorials
+- ros-melodic-tf2-msgs
+jenkins_job_label: buildagent_arm64
+jenkins_job_priority: 50
+jenkins_job_schedule: 15 23 * * *
+jenkins_job_timeout: 300
+package_selection_args: '--packages-up-to buildfarm_perf_tests --packages-skip performance_test buildfarm_perf_tests --packages-skip-regex .*connext.*'
+repos_files:
+- https://github.com/ros2/ros2/raw/master/ros2.repos
+- https://gist.githubusercontent.com/cottsay/d3ed9d3c7ed32165fdb987a09d434661/raw/ros2.repos
```

Comment:
I'm not sure what our policy is about this, maybe someone else from the team has an opinion on what to do here?

My suggestion would be to move @cottsay's changes to https://github.com/ros2/performance_test and use that until everything has been merged upstream into Apex's repo.

Reply:
I agree, let's fork the repo there until the upstream changes are merged. It is easy to switch later.

Reply:
Both of the repos are now under the `ros2` org. Hopefully Apex will merge our two PRs soon, which should eliminate one of them.

If we can get the packages listed in the official distribution, we can eliminate the repos file and just list the packages, which would be a lot cleaner.